### DISCUSSION TITLE
Curation report - week 18th of April

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/GPML-Repo#55] [Curation reports]
Hi

here is my weekly curation report:

46 pathways with changes.

@andrawaag  The description of https://www.wikipathways.org/index.php/Pathway:WP5216 mentions blue arrows for transcriptional stimulatory modification, but they are all black for me.

https://www.wikipathways.org/index.php/Pathway:WP820 shows an "Error rendering pathway page" after a change of title by Eweitz. (

exception 'MWException' with message 'Invalid pvjson, so cannot savePvjsonCache.' in /var/www/wikipathways/wpi/extensions/Pathways/Pathway.php:1278

)

I tried reverting to the previous version as suggested on the page, but that didn't solve the problem. Same issue can be seen on the same pathway from other species:

https://www.wikipathways.org/index.php/Pathway:WP938

https://www.wikipathways.org/index.php/Pathway:WP1172

However, the human and mouse pathways look fine, even though the same title change was performed: https://www.wikipathways.org/index.php/Pathway:WP558, https://www.wikipathways.org/index.php/Pathway:WP449

@egonw  In https://www.wikipathways.org/index.php/Pathway:WP3172, the pathway gastric pepsin release links to https://www.wikipathways.org/instance/WP2601, and Gastric Gastrin Release links to https://www.wikipathways.org/instance/WP2599, but these pathways don't exist. Also, the pathway is from Bos taurus, but Parietal Cell production links out to a human pathway. Is this intended? The remaining two Pathways in here look clickable, but are not.

https://www.wikipathways.org/index.php/Pathway:WP1344 links to 26S Proteasome Degradation, but the link just goes back to the original pathway again. Also here, Apoptosis links to the human pathway, not to Danio rerio.

I didn't check your added pathway links from other pathways, but maybe there is a problem with others too?

Search works.

Deleted one pathway (https://www.wikipathways.org/index.php/Pathway:WP5212, test pathway, delete was requested by author).

Next one is Friederike, but I don't see her in Github.

**Comment by khanspers** [comment]
Thanks for the report @danidi ! 

For the pathways with pvjson errors, I tried  manually replacing the gpml on the wiki with an older version, but it didn't work. Don't know what's going on. 
I propose that we replace the broken pathways with the [recently homology converted pathways](https://github.com/wikipathways/wikipathways-homology/tree/main/pathways) (by Helena). I did this for WP938 (Pan troglodytes) already and it now renders again. Unless there are objections, I will do this also for WP1172 (Canis familiaris). For Gallus gallus (WP820) there is no new gpml, presumably because the mapping threshold wasn't met. I propose to delete this one, unless someone can figure out how to fix the error. 

Regarding pathway links going from non-human pathways to human pathways, that's expected because of homology conversion. Only the gene/protein data nodes are converted, so pathway nodes will still be the same as the original human pathway.

Kristina

**Comment by egonw** [comment]
> @egonw In https://www.wikipathways.org/index.php/Pathway:WP3172, the pathway gastric pepsin release links to https://www.wikipathways.org/instance/WP2601, and Gastric Gastrin Release links to https://www.wikipathways.org/instance/WP2599, but these pathways don't exist. Also, the pathway is from Bos taurus, but Parietal Cell production links out to a human pathway. Is this intended? The remaining two Pathways in here look clickable, but are not.

Interesting. Some time ago I created a test to detect links to deleted pathways. Apparently, these instances are not picked up properly.

**Reply by egonw** [reply]
To be continued here: https://github.com/wikipathways/WikiPathwaysCurator/issues/22

**Comment by khanspers** [comment]
OK, I updated the Cf pathway also to the latest converted content. Before I delete [WP820](https://www.wikipathways.org/index.php/Pathway:WP820), @ariutta can you take a quick look at the pvjson error message and see if you see something obvious that we can fix?

**Comment by ariutta** [comment]
I took a look and didn't see an obvious answer, but I think I remember
seeing this before. I'll try running the converter locally tomorrow to see
what I get.

On Mon, Apr 25, 2022 at 9:18 PM Kristina Hanspers ***@***.***>
wrote:

> OK, I updated the Cf pathway also to the latest converted content. Before
> I delete WP820 <https://www.wikipathways.org/index.php/Pathway:WP820>,
> @ariutta <https://github.com/ariutta> can you take a quick look at the
> pvjson error message and see if you see something obvious that we can fix?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wikipathways/wikipathways-development/discussions/55#discussioncomment-2635701>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJ4TA4B7MXFSE6S3NN2G5DVG5VB7ANCNFSM5UCOV37Q>
> .
> You are receiving this because you were mentioned.Message ID:
> <wikipathways/wikipathways-development/repo-discussions/55/comments/2635701
> @github.com>
>


-- 
Anders Riutta
***@***.*** | Bioinformatics Core | Institute of Data
Science and Biotechnology
Gladstone Institutes | *Science Overcoming Disease*
